sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -96,23 +96,37 @@ NOTE: We shall consider adding other interoperability as needed, if possible.
 ### 4.4 Level-Zero handles' ownership and thread-safety
         
 The Level-Zero runtime doesn't do reference-counting of its objects, so it is crucial to adhere to these
-practices of how Level-Zero handles are manged.
+practices of how Level-Zero handles are manged. By default, the ownership us transferred to SYCL RT, but
+some interoparability API supports overriding this behavior and keep the ownership in the application.
+Use this enumeration for explicit specification of the ownership:
+``` C++
+enum ownership { transfer, keep };
+```
                 
-#### 4.4.1 SYCL runtime takes ownership
+#### 4.4.1 SYCL runtime takes ownership (default)
                 
 Whenever the application creates a SYCL object from the corresponding Level-Zero handle via one of the ```make<T>()``` functions,
-the SYCL runtime takes ownership of the Level-Zero handle. The application must not use the Level-Zero handle after
-the last host copy of the SYCL object is destroyed (as described in the core SYCL specification under
-"Common reference semantics"), and the application must not destroy the Level-Zero handle itself.
+the SYCL runtime takes ownership of the Level-Zero handle, if no explicit ```ownership::keep``` was specified.
+The application must not use the Level-Zero handle after the last host copy of the SYCL object is destroyed (
+as described in the core SYCL specification under "Common reference semantics"), and the application must not
+destroy the Level-Zero handle itself.
                                                                 
-#### 4.4.2 SYCL runtime assumes ownership
+#### 4.4.2 SYCL runtime assumes ownership (default)
 
 The application may call the ```get_native<T>()``` member function of a SYCL object to retrieve the underlying Level-Zero handle,
-however, the SYCL runtime continues to retain ownership of this handle. The application must not use this handle after
-the last host copy of the SYCL object is destroyed (as described in the core SYCL specification under
-"Common reference semantics"), and the application must not destroy the Level-Zero handle.
+however, the SYCL runtime continues to retain ownership of this handle, unless SYCL object was created with interoperability API that asked
+to keep the ownership by the application with ```ownership::keep```. The application must not use this handle after the last host copy of
+the SYCL object is destroyed (as described in the core SYCL specification under "Common reference semantics"), and the application must
+not destroy the Level-Zero handle.

```

Comment:
I would move this section below "Application keeps ownership (explicit)".  I would also rename the section title to match what is really being described here.  So, the outline would become:

* 4.4.1 SYCL runtime takes ownership (default)
* 4.4.2 Application keeps ownership (explicit)
* 4.4.3 Obtaining native handle does not change ownership

I would also change the text of this section to focus on the fact that the ownership is not changed when calling `get_native()`.  For example:

```
The application may call the ```get_native<T>()``` member function of a SYCL object to retrieve the underlying Level-Zero handle.
Doing so does not change the ownership of the the Level-Zero handle.  Therefore, the application may not use this
handle after the last host copy of the SYCL object is destroyed (as described in the core SYCL specification under
"Common reference semantics") unless the SYCL object was created by the application with ```ownership::keep```.

Reply:
Thanks, I think this is a good suggestion. Will follow